ext/reflection/tests/ReflectionEnum_getCase.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+ReflectionEnum::getCases()
+--FILE--
+<?php
+
+enum Foo {
+    case Bar;
+    const Baz = self::Bar;
+}
+
+$reflectionEnum = new ReflectionEnum(Foo::class);
+
+var_dump($reflectionEnum->getCase('Bar'));
+var_dump($reflectionEnum->getCase('Baz'));
+var_dump($reflectionEnum->getCase('Qux'));
+
+?>
+--EXPECT--
+bool(false)
+object(ReflectionEnumCase)#2 (2) {
+  ["name"]=>
+  string(3) "Baz"
```

Comment:
This is a bug in the implementation, see my other comment. The `case Bar` should have a ReflectionEnumCase, Baz shouldn't.
It would also be useful to add a check that `const Other = 1;` also causes getCase to return false